spec.html

```diff
@@ -968,6 +968,23 @@ <h1>
           1. Return _asyncContextMapping_.
         </emu-alg>
       </emu-clause>
+
+      <emu-clause id="sec-toasynccontextsnapshotobject" type="abstract operation">
+        <h1>
+          ToAsyncContextSnapshotObject (
```

Comment:
Just a minor name bikeshed: I'd interpret this as an alternative to the constructor rather than type conversion.

```suggestion
          CreateAsyncContextSnapshot (
```